presto-verifier/src/main/java/com/facebook/presto/verifier/framework/QueryConfigurationOverrides.java

```diff
@@ -13,15 +13,27 @@
  */
 package com.facebook.presto.verifier.framework;
 
+import java.util.Map;
 import java.util.Optional;
 
 public interface QueryConfigurationOverrides
 {
+    enum SessionPropertiesOverrideStrategy
+    {
+        NO_OVERRIDE,
```

Comment:
Theoretically `SUBSTITUTE` is also `NO_OVERRIDE`. Maybe something like `NO_ACTION`?